scout/adapter/mongo/case.py

```diff
@@ -288,28 +297,29 @@ def _update_case_id_query(self, query, id_list):
 
     def cases(
         self,
-        owner=None,
-        collaborator=None,
-        query=None,
-        skip_assigned=False,
-        has_causatives=False,
-        reruns=False,
-        rerun_monitor=False,
-        finished=False,
-        research_requested=False,
-        is_research=False,
-        has_rna_data=False,
-        status=None,
-        phenotype_terms=False,
-        group=None,
-        pinned=False,
-        cohort=False,
-        name_query=None,
-        yield_query=False,
-        within_days=None,
-        assignee=None,
-        verification_pending=None,
-        has_clinvar_submission=None,
+        owner: str = None,
```

Comment:
💯 Perhaps add Optional where the key is optional )most of them I guess?)

Reply:
It's Optional with a capital O, mind you!